lib/iex/lib/iex/server.ex

```diff
@@ -82,7 +82,8 @@ defmodule IEx.Server do
               {^ref, nil} ->
                 {:error, :refused}
               {^ref, leader} ->
-                IEx.Evaluator.start(server, leader)
+                {_, evaluator_state} = run_state(opts)
```

Comment:
That's true, I wasn't sure if this was OK but figured you'd mention if it wasn't. :)